pip/utils/__init__.py

```diff
@@ -3,7 +3,9 @@
 import contextlib
 import errno
 import locale
-import logging
+# we have a submodule named 'logging' which would shadow this if we used the
+# regular name:
+import logging as std_logging
```

Comment:
I'm confused why this change is needed. You don't seem to be using the name ``logging`` anywhere except for the stdlib logging. There is a ``pip.utils.logging`` module, but that shouldn't cause any problems.